seen on master during `make run` in CI

Prior to submitting an issue, check to see if one has already been created. If there is currently an open issue, add a thumbs-up emoji to identify that it is also affecting you.

### Your Environment

- **Version:** ubuntu VM 
- **Affected Component:** CI
- **Affected Subcomponent:** Orchestrator pod/service or AGW service
- **Deployment Environment:** e.g. AWS (orc8r), local docker-compose (orc8r), bare-metal (AGW), Vagrant (AGW)

### Repro Steps
```
cd magma/lte/gateway
fab integ_test
```

### Describe the Issue
Seeing this in CI
```
[magma@10.240.0.33] out: [vagrant@127.0.0.1:2222] out: Building with sentry!
[magma@10.240.0.33] out: [vagrant@127.0.0.1:2222] out: -- Configuring done
[magma@10.240.0.33] out: [vagrant@127.0.0.1:2222] out: -- Generating done
[magma@10.240.0.33] out: [vagrant@127.0.0.1:2222] out: -- Build files have been written to: /home/vagrant/build/c/session_manager
[magma@10.240.0.33] out: [vagrant@127.0.0.1:2222] out: # Turn on generation of compile_commands.json (See GH Issue #5488)
[magma@10.240.0.33] out: [vagrant@127.0.0.1:2222] out: cd  /home/vagrant/build/c/session_manager && awk '{if (/^CMAKE_EXPORT_COMPILE_COMMANDS/) gsub(/OFF/, "ON"); print}' CMakeCache.txt > tmp && mv tmp CMakeCache.txt
[magma@10.240.0.33] out: [vagrant@127.0.0.1:2222] out: ninja -C  /home/vagrant/build/c/session_manager
[magma@10.240.0.33] out: [vagrant@127.0.0.1:2222] out: ninja: Entering directory `/home/vagrant/build/c/session_manager'
[magma@10.240.0.33] out: [vagrant@127.0.0.1:2222] out: 
[magma@10.240.0.33] out: [vagrant@127.0.0.1:2222] out: [0/1] Re-running CMake...
[magma@10.240.0.33] out: [vagrant@127.0.0.1:2222] out: Found CLANG_FORMAT: /usr/bin/clang-format
[magma@10.240.0.33] out: [vagrant@127.0.0.1:2222] out: Adding code coverage build and linker flags for BUILD_TESTS
[magma@10.240.0.33] out: [vagrant@127.0.0.1:2222] out: Build type is Debug
[magma@10.240.0.33] out: [vagrant@127.0.0.1:2222] out: Async Proto_srcs are /home/vagrant/build/c/session_manager/datastore/lte/protos/session_manager.pb.cc;/home/vagrant/build/c/session_manager/datastore/lte/protos/policydb.pb.cc;/home/vagrant/build/c/session_manager/datastore/lte/protos/mobilityd.pb.cc;/home/vagrant/build/c/session_manager/datastore/lte/protos/apn.pb.cc;/home/vagrant/build/c/session_manager/datastore/lte/protos/subscriberdb.pb.cc;/home/vagrant/build/c/session_manager/datastore/lte/protos/session_manager.grpc.pb.cc;/home/vagrant/build/c/session_manager/datastore/orc8r/protos/common.pb.cc;/home/vagrant/build/c/session_manager/datastore/orc8r/protos/redis.pb.cc
[magma@10.240.0.33] out: [vagrant@127.0.0.1:2222] out: Proto_srcs are /home/vagrant/build/c/session_manager/policydb/orc8r/protos/common.pb.cc;/home/vagrant/build/c/session_manager/policydb/lte/protos/policydb.pb.cc
[magma@10.240.0.33] out: [vagrant@127.0.0.1:2222] out: Proto_srcs are /home/vagrant/build/c/session_manager/orc8r/protos/common.pb.cc;/home/vagrant/build/c/session_manager/orc8r/protos/directoryd.pb.cc;/home/vagrant/build/c/session_manager/lte/protos/session_manager.pb.cc;/home/vagrant/build/c/session_manager/lte/protos/apn.pb.cc;/home/vagrant/build/c/session_manager/lte/protos/subscriberdb.pb.cc;/home/vagrant/build/c/session_manager/lte/protos/policydb.pb.cc;/home/vagrant/build/c/session_manager/lte/protos/pipelined.pb.cc;/home/vagrant/build/c/session_manager/lte/protos/spgw_service.pb.cc;/home/vagrant/build/c/session_manager/lte/protos/mconfig/mconfigs.pb.cc;/home/vagrant/build/c/session_manager/feg/gateway/services/aaa/protos/accounting.pb.cc;/home/vagrant/build/c/session_manager/feg/gateway/services/aaa/protos/context.pb.cc;/home/vagrant/build/c/session_manager/orc8r/protos/directoryd.grpc.pb.cc;/home/vagrant/build/c/session_manager/lte/protos/session_manager.grpc.pb.cc;/home/vagrant/build/c/session_manager/lte/protos/pipelined.grpc.pb.cc;/home/vagrant/build/c/session_manager/lte/protos/spgw_service.grpc.pb.cc;/home/vagrant/build/c/session_manager/feg/gateway/services/aaa/protos/accounting.grpc.pb.cc
[magma@10.240.0.33] out: [vagrant@127.0.0.1:2222] out: Building with sentry!
[magma@10.240.0.33] out: [vagrant@127.0.0.1:2222] out: -- Configuring done
[magma@10.240.0.33] out: [vagrant@127.0.0.1:2222] out: -- Generating done
[magma@10.240.0.33] out: [vagrant@127.0.0.1:2222] out: -- Build files have been written to: /home/vagrant/build/c/session_manager
[magma@10.240.0.33] out: [vagrant@127.0.0.1:2222] out: ninja: error: dependency cycle: datastore/libDATASTORE.a -> datastore/CMakeFiles/DATASTORE.dir/lte/protos/session_manager.pb.cc.o -> lte/protos/session_manager.pb.h -> datastore/libDATASTORE.a
[magma@10.240.0.33] out: [vagrant@127.0.0.1:2222] out: make: *** [Makefile:185: build_session_manager] Error 1
[magma@10.240.0.33] out: [vagrant@127.0.0.1:2222] out: 
[magma@10.240.0.33] out: 
[magma@10.240.0.33] out: 
[magma@10.240.0.33] out: Fatal error: run() received nonzero return code 2 while executing!
[magma@10.240.0.33] out: 
[magma@10.240.0.33] out: Requested: make run
[magma@10.240.0.33] out: Executed: /bin/bash -l -c "cd \$MAGMA_ROOT/lte/gateway >/dev/null && make run"
```

--- a/lte/gateway/c/session_manager/CMakeLists.txt
+++ b/lte/gateway/c/session_manager/CMakeLists.txt
@@ -79,11 +79,11 @@ create_proto_dir("feg/gateway/services/aaa" CWF_CPP_OUT_DIR)
 list(APPEND PROTO_SRCS "")
 list(APPEND PROTO_HDRS "")
 
-set(SMGR_ORC8R_CPP_PROTOS common directoryd)
+set(SMGR_ORC8R_CPP_PROTOS directoryd)
 generate_cpp_protos("${SMGR_ORC8R_CPP_PROTOS}" "${PROTO_SRCS}"
   "${PROTO_HDRS}" ${ORC8R_PROTO_DIR} ${ORC8R_CPP_OUT_DIR})
 
-set(SMGR_LTE_CPP_PROTOS session_manager apn subscriberdb policydb
+set(SMGR_LTE_CPP_PROTOS session_manager
   pipelined spgw_service mconfig/mconfigs)
 generate_cpp_protos("${SMGR_LTE_CPP_PROTOS}" "${PROTO_SRCS}"
   "${PROTO_HDRS}" ${LTE_PROTO_DIR} ${LTE_CPP_OUT_DIR})
@@ -104,7 +104,7 @@ set(SMGR_CWF_GRPC_PROTOS accounting)
 generate_grpc_protos("${SMGR_CWF_GRPC_PROTOS}" "${PROTO_SRCS}"
   "${PROTO_HDRS}" ${CWF_PROTO_DIR} ${CWF_CPP_OUT_DIR})
 
-message("Proto_srcs are ${PROTO_SRCS}")
+message("SessionManager proto_srcs are ${PROTO_SRCS}")
 
 link_directories(
   ${MAGMA_LIB_DIR}/async_grpc

--- a/lte/gateway/c/session_manager/datastore/CMakeLists.txt
+++ b/lte/gateway/c/session_manager/datastore/CMakeLists.txt
@@ -21,16 +21,16 @@ include($ENV{MAGMA_ROOT}/orc8r/gateway/c/common/CMakeProtoMacros.txt)
 list(APPEND PROTO_SRCS "")
 list(APPEND PROTO_HDRS "")
 
-set(ASYNC_ORC8R_CPP_PROTOS common redis)
-set(ASYNC_LTE_CPP_PROTOS "")
-set(ASYNC_LTE_GRPC_PROTOS "")
-set(ASYNC_ORC8R_GRPC_PROTOS "")
+set(DS_ORC8R_CPP_PROTOS common redis)
+set(DS_LTE_CPP_PROTOS "")
+set(DS_LTE_GRPC_PROTOS "")
+set(DS_ORC8R_GRPC_PROTOS "")
 
-generate_all_protos("${ASYNC_LTE_CPP_PROTOS}" "${ASYNC_ORC8R_CPP_PROTOS}"
-    "${ASYNC_LTE_GRPC_PROTOS}"
-    "${ASYNC_ORC8R_GRPC_PROTOS}" "${PROTO_SRCS}" "${PROTO_HDRS}")
+generate_all_protos("${DS_LTE_CPP_PROTOS}" "${DS_ORC8R_CPP_PROTOS}"
+    "${DS_LTE_GRPC_PROTOS}"
+    "${DS_ORC8R_GRPC_PROTOS}" "${PROTO_SRCS}" "${PROTO_HDRS}")
 
-message("Async Proto_srcs are ${PROTO_SRCS}")
+message("Datastore Proto_srcs are ${PROTO_SRCS}")
 
 add_library(DATASTORE
     ObjectMap.h

--- a/lte/gateway/c/session_manager/policydb/CMakeLists.txt
+++ b/lte/gateway/c/session_manager/policydb/CMakeLists.txt
@@ -28,11 +28,11 @@ set(POLICYDB_ORC8R_CPP_PROTOS common)
 generate_cpp_protos("${POLICYDB_ORC8R_CPP_PROTOS}" "${PROTO_SRCS}"
     "${PROTO_HDRS}" ${ORC8R_PROTO_DIR} ${ORC8R_CPP_OUT_DIR})
 
-set(POLICYDB_LTE_CPP_PROTOS policydb)
+set(POLICYDB_LTE_CPP_PROTOS policydb apn mobilityd subscriberdb)
 generate_cpp_protos("${POLICYDB_LTE_CPP_PROTOS}" "${PROTO_SRCS}"
     "${PROTO_HDRS}" ${LTE_PROTO_DIR} ${LTE_CPP_OUT_DIR})
 
-message("Proto_srcs are ${PROTO_SRCS}")
+message("Policydb proto_srcs are ${PROTO_SRCS}")
 
 add_library(POLICYDB
     PolicyLoader.cpp

